Tests/SvgW3CTestRunner/BitmapExtensions.cs

```diff
@@ -61,4 +62,120 @@ public int Reserved
             }
         }
     }
+
+    /// <summary>
+    /// Taken from https://web.archive.org/web/20130111215043/http://www.switchonthecode.com/tutorials/csharp-tutorial-convert-a-color-image-to-grayscale
+    /// and slightly modified.
+    /// Image width and height, default threshold and handling of alpha values have been adapted.
+    /// </summary>
+    static class ExtensionMethods
+    {
+        private static readonly int ImageWidth = 64;
+        private static readonly int ImageHeight = 64;
+
+        public static float PercentageDifference(this Image img1, Image img2, byte threshold = 10)
```

Comment:
I understand this is copied over from `ImageComparisonTest`. I wonder if it makes  sense to move this into a common component to avoid the duplication (and potential bugs if the code is changed in ne place only).

Reply:
It makes sense as the shared parts are increasing. What name will you suggest for the shared code?
- Svg.Tests.Common
- Svg.Tests.Shared
- Svg.Tests.Core
- etc

Reply:
`Common` sounds fine, but I have no strong feelings here.

Reply:
Thanks, will look into it - I will be leaving for church service, and they also requested some IT services from me so will not be available for at least 8 hours.

Reply:
@mrbean-bremen After looking into it, we should consider the shared library as separate PR.
Currently, the `Svg.Benchmark` project references the `Svg.UnitTests` project just to access an extension method, and I think the shared library should extend there too.
As a separate PR, we will give it a better review.

```xml
    <ItemGroup>
        <ProjectReference Include="..\..\Source\Svg.csproj" />
        <ProjectReference Include="..\Svg.UnitTests\Svg.UnitTests.csproj" />
    </ItemGroup>
```

Reply:
Sure, agreed!

Reply:
Thanks, if there is no remaining issue on this PR, can I proceed with the merge?